packages/engine/src/simulation/package/context/packages/agent_messages/package.js

```diff
@@ -1,3 +1,5 @@
+import * as hash_util from "./src/worker/runner/javascript/hash_util.js";
```

Comment:
I wonder if we want to import only the symbols needed to be explicit.

Reply:
I didn't import them individually because the file uses `hash_util.<function>` and I didn't want to change things in JS files more than needed.
But I can change that in this file and others.